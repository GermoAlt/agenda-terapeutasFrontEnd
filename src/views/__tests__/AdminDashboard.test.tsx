// @ts-ignore
import React from 'react';
import AdminDashboard from "../AdminDashboard";
import {render} from "@testing-library/react";

it('renders according to snapshot', () => {
    const component = render(<AdminDashboard />);
    expect(component.baseElement).toMatchSnapshot();
});