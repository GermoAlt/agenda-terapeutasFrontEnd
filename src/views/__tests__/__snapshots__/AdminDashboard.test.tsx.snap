// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders according to snapshot 1`] = `
<body>
  <div>
    <h1>
      Admin Dashboard
    </h1>
  </div>
</body>
`;
